experience.css

```diff
@@ -1,55 +1,119 @@
+@import url(./global.css);
+@import url("https://fonts.googleapis.com/css?family=Dosis:200,400&display=swap");
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+#page-title {
+  text-align: center;
+  margin: 20px;
+}
+
 .article-experience {
-  padding: 0.3rem;
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+  align-items: center;
+  margin: 50px;
+  justify-content: center;
+  gap: 20px;
+}
+#experience {
+  display: flex;
+  margin: 10px;
+}
+.experience {
+  height: 310px;
+  width: 250px;
   background-color: var(--background-color-nav);
-  margin-left: 5px;
-  margin-right: 10px;
-  display: inline-block;
+  color: var(--);
+  padding: 10px;
+  border-radius: 10px;
+
+  box-shadow: rgb(38, 57, 77) 0px 20px 30px -10px;
 }
 
-#experience {
-  margin: 0.5rem;
-  padding: 0.3rem;
-  font-size: 1.2rem;
-  background: white;
+.experience:hover {
+  height: 320px;
+  width: 260px;
 }
 
-.article-studies {
-  padding: 0.3rem;
-  background-color: var(--background-color-nav);
-  margin-right: 5px;
-  display: inline-block;
+/* Read more/ Read less */
+.experience .open {
+  max-height: 50px;
+  transition: max-height 0.3s cubic-bezier(0.9, 0, 0.8, 0.2);
 }
 
-#studies {
-  margin: 0.5rem;
-  padding: 0.3rem;
-  font-size: 1.2rem;
-  background: white;
+@keyframes open {
+  from {
+    line-clamp: 3;
+    -webkit-line-clamp: 3;
+  }
+  to {
+    line-clamp: initial;
+    -webkit-line-clamp: initial;
+  }
 }
 
-footer {
-  display: block;
-  margin-top: 50px;
+@keyframes close {
+  from {
+    line-clamp: initial;
+    -webkit-line-clamp: initial;
+  }
+  to {
+    line-clamp: 3;
+    -webkit-line-clamp: 3;
+  }
 }
 
-body {
-  align-items: center;
-  text-align: center;
+.description-experience {
+  display: -webkit-box;
```

Comment:
This is deprecated [please check](https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient). We need to find an alternative way of achieving the same behaviour.